.github/workflows/docker-ghcr.yml

```diff
@@ -21,7 +21,7 @@ on:
 
 env:
   registry: ghcr.io
-  username: ${{ github.actor }}
+  username: ${{ github.repository_owner }}
```

Comment:
Given that `dockerhub-scope` variable is being used in `docker-dockerhub.yml`, we might as well use `ghcr-scope` as variable name.

Reply:
Dockerhub-scope is a workaround to let users push to an organization they're part of. Unlike DockerHub, GitHub already allows orgs to publish their own packages so I believe ghcr-scope would not be necessary in that regard specifically.

Additionally, we're using the automatic GitHub Token that is [scoped to the repository itself](https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#about-the-github_token-secret). To allow a user's repo to push to an org's repo, the user would need to add another secret for a custom GitHub Token. It won't be much code in the workflow but it could be extra overhead for the user.

The way it works presently lets users publish to their GitHub Packages without any setup at all, which is pretty nice to have IMO.

Reply:
The existing setup is nice. I am only suggesting variable name change on the left-side of line-24. Everything else is same as you provided. The suggestion is to improve readability.

Reply:
Ahh gotcha, will do